modules/ROOT/pages/bat-reporting-task.adoc

```diff
@@ -233,4 +234,20 @@ And the result is:
 {
   "result": "The suite <Hello world suite> with assertion <answer 200> is ok"
 }
-----
\ No newline at end of file
+----
+
+== Email
+
+When using the Anypoint Platform you can create an email reporter to enable email alerts to several accounts.
```

Comment:
What do you mean by "when using the Anypoint Platform"? Using it in what way? 
Do the listed email addresses need to be associated with accounts in Anypoint Platform?